openapi.yaml

```diff
@@ -2372,10 +2485,80 @@ components:
           allOf:
             - $ref: "#/components/schemas/IdentityVerificationLinks"
 
+    IdentityVerification:
+      type: object
+      required: [id]
+      properties:
+        id:
+          readOnly: true
+          allOf:
+            - $ref: "#/components/schemas/IdentityVerificationId"
+      allOf:
+        - $ref: "#/components/schemas/DatedObject"
+        - $ref: "#/components/schemas/IdentityVerificationBase"
+
+    IdentityVerificationOutput:
+      type: object
+      required: [id]
+      properties:
+        id:
+          $ref: "#/components/schemas/IdentityVerificationId"
+      allOf:
+        - $ref: "#/components/schemas/DatedObject"
+        - $ref: "#/components/schemas/IdentityVerificationCreateBase"
+
+    IdentityVerificationBase:
+      type: object
+      required:
+        [applicant_id, declared_data, status, webhook_url, _links]
```

Comment:
redirect_url is removed. It is already the new-idv endpoint description

Reply:
removed